magefile.go

```diff
@@ -46,7 +46,7 @@ var (
 	pythonExecutableName = "python3"
 	ttExecutableName     = "tt"
 
-	generateModePath = filepath.Join(packagePath, "codegen", "generate_code.go")
+	generateModePath = "./" + filepath.Join(packagePath, "codegen", "generate_code.go")
```

Comment:
Add a description that `filepath.Join()` calls `filepath.Clean()` and removes `./` and the reason why `./cli/some_path` does not equal `cli/some_path`.

Reply:
added